# WpGen.srv

---
CropLine left_line
CropLine right_line